app/components/emoji/emoji.js

```diff
@@ -49,10 +64,16 @@ export default class Emoji extends React.PureComponent {
             return <Text style={textStyle}>{literal}</Text>;
         }
 
+        let ImageComponent = FastImage;
+        if (Platform.OS === 'android') {
```

Comment:
So the short answer yes we can and we should for all image components that use network images going forward. In the case of emojis though, we can't. When I try to use it for an emoji on Android we get the `nested View in a Text node` error. Still waiting on RN to fix that one.